wandb/sdk/lib/tracelog.py

```diff
@@ -46,6 +46,10 @@
 
 ANNOTATE_QUEUE_NAME = "_DEBUGLOG_QUEUE_NAME"
 
+# capture stdout and stderr before anyone messes with them
+stdout_write = sys.__stdout__.write
+stderr_write = sys.__stderr__.write
```

Comment:
I am not sure why this was not needed with the old console wrap.
but it seems like the correct fix for a tracelog debug output